vpc-classiclink/recipes/install.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook Name:: vpc-classiclink
+# Recipe:: install
+#
+include_recipe 'python::pip'
+
+python_pip "awscli" do
```

Comment:
Please re-use the existing cookbook:
https://github.com/till/easybib-cookbooks/tree/master/awscli